### PR TITLE
Issue 232: EMT Grid Following corrections

### PR DESCRIPTION
**Pull request by Carlos-Alegre**
Pull request for the EMT Grid Following VSC corrections

### CODE REVIEW

File: pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md [hunk 1]
```diff
@@ -310,181 +310,64 @@ $$K^{ipl}_i = \frac{2}{3V_{peak}\tau_p}$$
...
-<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
-<img src="{{ '/pages/models/generations/Sources/VSC/EMTGridFollowingVSC/SVPWM_figs.svg' | relative_url }}"
-     alt="SVPWM Switching State"
-     style="float: center; margin-right: 10px; width: 900px;" />
-</div>
-<div align = 'center'>
-Figure 10: Schematics of the instantaneous switching state ⑥. (a) shows the positions of the switches, as well as the whole pattern, (b) represents the IGBT pairs and which of them is on, and (c) shows the equivalent grid for the state.
-</div>
-<br>
+## Parameter tunning
```
Comment (Lampros-Cresym) [inline pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md:351]:
```suggestion
## Parameter tuning 
```

File: pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md [hunk 2]
```diff
@@ -310,181 +310,64 @@ $$K^{ipl}_i = \frac{2}{3V_{peak}\tau_p}$$
...
-     style="float: center; margin-right: 10px; width: 900px;" />
-</div>
-<div align = 'center'>
-Figure 10: Schematics of the instantaneous switching state ⑥. (a) shows the positions of the switches, as well as the whole pattern, (b) represents the IGBT pairs and which of them is on, and (c) shows the equivalent grid for the state.
-</div>
-<br>
+## Parameter tunning 
 
-As it can be seen in Figure 10(a), the pattern needed to generate the $$v^{\alpha\beta}$$ AC voltage shown in Figure 9 will change the state of one of its switches at a time and starts and ends at 0 voltage. The states $$000$$ and $$111$$ are those where all the switches are closed on the same side, short-circuiting the three phases, while for the rest of the cases, there is a voltage division between the isolated phase and the short-circuited couple, as can be seen from the grid equivalent in Figure 10(c).
+The following table shows possible values for the parameters of the controllers using the tunning proposed:
```
Comment (Lampros-Cresym) [inline pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md:353]:
```suggestion
The following table shows possible values for the parameters of the controllers using the tuning proposed:
```

File: pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md [hunk 3]
```diff
@@ -310,181 +310,64 @@ $$K^{ipl}_i = \frac{2}{3V_{peak}\tau_p}$$
...
-
-<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
-
-$$ v_{c}^{\alpha\beta} = D_1 V_1 + D_2 V_2$$
-</div>
-
-where, for the given example, $$V_1 = E_{DC} e^{j0}$$ and $$V_2 = E_{DC} e^{j\frac{\pi}{3}}$$, and $$D_1$$ and $$D_2$$ are the duty cycles calculated for the given voltage setpoint. The output voltage can be then transformed to the *abc* frame.
+| Parameter | Value | Units |
+| --------- | ----- | ----- |
+| $$V_{peak}$$ | $$2500$$ | V |
```
Comment (Lampros-Cresym) [inline pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md:357]: where is V_peak?

Comment (Lampros-Cresym) [reply pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md:357]: Ok, I guess it was there before but has to be removed along with the SPWM part

**Review by Lampros-Cresym — COMMENTED** [review]
That is fine. Could you please just make sure that all references (figures, equations, etc.) are working appropriately?

File: pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md (hunk 3 above)
Comment (Carlos-Alegre) [reply pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md]: Removed Vpeak. All the references and Figures work ok for me

**Review by GJCRESYM — APPROVED** [review]

**Review by Lampros-Cresym — APPROVED** [review]

**Pull request merged**